core/io/config_file.h

```diff
@@ -61,7 +67,7 @@ class ConfigFile : public RefCounted {
 	void erase_section(const String &p_section);
 	void erase_section_key(const String &p_section, const String &p_key);
 
-	Error save(const String &p_path);
+	Error save(const String &p_path, FileAccess::SaveIntegrityLevel p_integrity_level = FileAccess::SAVE_INTEGRITY_DEFAULT);
```

Comment:
There are also `save_encrypted()` and `save_encrypted_pass()`.

Reply:
Done.